report_xls/__openerp__.py

```diff
@@ -26,8 +26,7 @@
     'author': "Noviat,Odoo Community Association (OCA)",
     'website': 'http://www.noviat.com',
     'category': 'Reporting',
-    'description': """
-Excel report engine
+    'description': """Excel report engine
```

Comment:
You can cherry-commit https://github.com/OCA/reporting-engine/commit/fddca415b8d255b9d877ea92217ea086205ce077 from 8.0